Content.Server/GameObjects/EntitySystems/ListeningSystem.cs

```diff
@@ -15,23 +16,23 @@ class ListeningSystem : EntitySystem
     {
 #pragma warning disable 649
         [Dependency] private readonly IMapManager _mapManager;
-        [Dependency] private readonly IEntitySystemManager _entitySystemManager;
 #pragma warning restore 649
 
         public override void Initialize()
         {
             base.Initialize();
-            EntityQuery = new TypeEntityQuery(typeof(ListeningComponent));
         }
 
         public void PingListeners(IEntity source, GridCoordinates sourcePos, string message)
         {
-            foreach (var listener in RelevantEntities)
+            foreach (var listener in ComponentManager.EntityQuery<IListen>())
             {
-                var dist = sourcePos.Distance(_mapManager, listener.Transform.GridPosition);
-
-                listener.GetComponent<ListeningComponent>()
-                        .PassSpeechData(message, source, dist);
+                var listenerPos = listener.GetListenerPosition();
```

Comment:
As above just use listener.Owner. This should also be MapPosition instead and pretty much everything should be MapPosition as GridPosition is relative to the grid so cross-grid comms might be sketchy.